app.go

```diff
@@ -88,6 +87,26 @@ func (a *App) Start(torConfig *data.TorConfig) error {
 		}
 	}
 
+	// a gorutine that will call ensurelatestBackup when lnnode is ready
+	go func() {
```

Comment:
Do we need another go routine to subscribe for daemon events? the call to watchDaemonEvents at the bottom already subscribes so perhaps it is better to hook the ensureLatestBackup verification there?

Reply:
No, good point. I moved it to `watchDaemonEvents` now.